cluster/gce/util.sh

```diff
@@ -848,6 +848,7 @@ function create-master() {
   # from the other cluster variables so that the client (this
   # computer) can forget it later. This should disappear with
   # http://issue.k8s.io/3168
+  KUBE_CONTROLLER_MANAGER_TOKEN=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
```

Comment:
Would it be possible to move this to be generated on the master? In GCE/GKE, the controller-manager is always on the same host as the apiserver.